utils/config.go

```diff
@@ -668,7 +664,6 @@ func GenerateLimitedClientConfig(c *model.Config, diagnosticId string, license *
 	props["LdapLoginButtonColor"] = ""
 	props["LdapLoginButtonBorderColor"] = ""
 	props["LdapLoginButtonTextColor"] = ""
-	props["EnableMultifactorAuthentication"] = "false"
```

Comment:
This was a duplicate, it gets set here https://github.com/mattermost/mattermost-server/blob/e1dc4cc132bf620420492483b503eb08bbff3420/utils/config.go#L677

Reply:
That probably got left over when we removed the license requirement from it